tutoraspects/asset_command_helpers.py

```diff
@@ -161,7 +161,7 @@ class ChartAsset(Asset):
         "params.datasource",
         "params.slice_id",
     ]
-    raw_vars = ["sqlExpression", "query_context"]
+    raw_vars = ["query_context"]
```

Comment:
Will this break translated dataset strings? It was added in that PR.

Reply:
Hmm, yeah I didn't consider that. I will restore it but we need to have in the map that we need to apply the Jinja template on the fields. At least now that the Couse Dashboard is done and the charts are setup we can work on translations and moving metrics from the charts to the datasets.

You can create an issue and I will take it, also the following PR will fix the None values on the translated datasets.